app/models/rental.py

```diff
@@ -0,0 +1,43 @@
+from flask import current_app
+from app import db
+from datetime import datetime, timedelta
+from app.models.customer import Customer
+from app.models.video import Video
+
+class Rental(db.Model): 
+    rental_id = db.Column(db.Integer, primary_key=True)
+    due_date = db.Column(db.DateTime, nullable=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey("customer.customer_id"), nullable=True)
+    video_id = db.Column(db.Integer, db.ForeignKey("video.video_id"), nullable=True)
+
```

Comment:
You may want to add a check in date to know if the user returned this rental:

```suggestion
    check_in_date = db.Column(db.DateTime, nullable=True)
```